assignments/arrays.js

```diff
@@ -1,96 +1,151 @@
+    {"id":26,"car_make":"Cadillac","car_model":"Escalade","car_year":2005},
+    {"id":27,"car_make":"Infiniti","car_model":"Q","car_year":2000},
+    {"id":28,"car_make":"Suzuki","car_model":"Aerio","car_year":2005},
+    {"id":29,"car_make":"Mercury","car_model":"Topaz","car_year":1993},
+    {"id":30,"car_make":"BMW","car_model":"6 Series","car_year":2010},
+    {"id":31,"car_make":"Pontiac","car_model":"GTO","car_year":1964},
+    {"id":32,"car_make":"Dodge","car_model":"Ram Van 3500","car_year":1999},
+    {"id":33,"car_make":"Jeep","car_model":"Wrangler","car_year":2011},
+    {"id":34,"car_make":"Ford","car_model":"Escort","car_year":1991},
+    {"id":35,"car_make":"Chrysler","car_model":"300M","car_year":2000},
+    {"id":36,"car_make":"Volvo","car_model":"XC70","car_year":2003},
+    {"id":37,"car_make":"Oldsmobile","car_model":"LSS","car_year":1997},
+    {"id":38,"car_make":"Toyota","car_model":"Camry","car_year":1992},
+    {"id":39,"car_make":"Ford","car_model":"Econoline E250","car_year":1998},
+    {"id":40,"car_make":"Lotus","car_model":"Evora","car_year":2012},
+    {"id":41,"car_make":"Ford","car_model":"Mustang","car_year":1965},
+    {"id":42,"car_make":"GMC","car_model":"Yukon","car_year":1996},
+    {"id":43,"car_make":"Mercedes-Benz","car_model":"R-Class","car_year":2009},
+    {"id":44,"car_make":"Audi","car_model":"Q7","car_year":2012},
+    {"id":45,"car_make":"Audi","car_model":"TT","car_year":2008},
+    {"id":46,"car_make":"Oldsmobile","car_model":"Ciera","car_year":1995},
+    {"id":47,"car_make":"Volkswagen","car_model":"Jetta","car_year":2007},
+    {"id":48,"car_make":"Dodge","car_model":"Magnum","car_year":2008},
+    {"id":49,"car_make":"Chrysler","car_model":"Sebring","car_year":1996},
+    {"id":50,"car_make":"Lincoln","car_model":"Town Car","car_year":1999}
+];
 // PROJECT RESTRICTION: You can't use map, reduce, or filter to solve these problems.  Only use native JavaScript for loops.
-
-// Example for loop:
-// arr = [1,2,3,4];
-// for (i = 0; i < arr.length; i++) {
-//     arr[i]; // 1,2,3,4
-// }
+// Sorting function
+function sortOn (arr, prop) {
+    arr.sort (
+        function (a, b) {
+            if (a[prop] < b[prop]){
+                return -1;
+            } else if (a[prop] > b[prop]){
+                return 1;
+            } else {
+                return 0;   
+            }
+        }
+    );
+}
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
+for (let i=0; i<inventory.length; i++) {
+    if (inventory[i].id === 33) {
+        console.log(`Car 33 is a ${inventory[i].car_year} ${inventory[i].car_make} ${inventory[i].car_model} \n-------------------------------\n`);
```

Comment:
```suggestion
        console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32]car_make} ${inventory[32].car_model}`);
```
Just another way to do it that's a little shorter without the for loop, but your way is more dynamic.